pages.ko/common/!.md

```diff
@@ -0,0 +1,24 @@
+# Exclamation mark
```

Comment:
```suggestion
# !
```

Reply:
Unfortunately, our parser doesn't support symbols/translations for the page title, so that's why we went with "Exclamation mark" in the initial PR too.

Reply:
I will open another PR to fix the title.

Reply:
> I will open another PR to fix the title.

That would probably get an error from tldr-bot too, we should add an exclusion for this in tldr-lint.

Reply:
Yes, I just noticed the English page uses `Exclamation mark` as well. 😅